gips/data/sentinel2/sentinel2.py

```diff
@@ -612,6 +613,11 @@ class sentinel2Data(Data):
             'bands': [{'name': band_name, 'units': Data._unitless}
                       for band_name in Asset._sensors['S2A']['indices-colors']],
         },
+        'cfmask': {
+            'description': 'Cloud mask',
+            'assets': [_asset_type],
+            'bands': ['cloudmask'],
```

Comment:
Needs to be made DH compliant.  `bands': {'name': 'cfmask', 'units': Data._unitless}` is probably the thing